### DISCUSSION TITLE
Moving to 4.x with modified theme files

### DISCUSSION THREAD

**Discussion by davidgs** [hugo-themes/toha#981] [Q&A]
First I want to say thank you for this amazing theme. I've been using it for years and it is fantastic! 

Having said that, moving to the v4 theme has been anything *but* fantastic. It's been a massive pain. In using Toha before, I made a fair number of modifications to the theme. Mostly by creating a separate theme dir `davidgs` and moving theme layouts, etc. that I wanted to modify there. My `config.yaml` then listed the `davidgs` theme *first* and the `toha` theme second. This worked great. 

With the move to a module-based theme, this has unleashed havoc. 

I modified `layouts/_default/single.html` in the old `toha` directory, but now that file is in the mod-cache, so I can't modify it without those changes being over-written. Similarly, I modified `/layouts/partials/helpers/get-author-image.html` with the same problem. (if you want to see the results of those modifications, check out https://davidgs.com/ and you'll see the "Author" icon in the center as it rotates through a series of images. :-) 

So, how do I proceed here?

**Comment by hossainemruz** [comment]
Hi @davidgs! You are not supposed to update the `layouts` files in the theme directory. Instead, you should copy the files that you want to modify and put it into similar path under `layouts` directory of your repo and modify it there. Hugo will use your modified layout files instead of the themes one.

Check this guide: https://toha-guides.netlify.app/posts/customizing/customize-layout/#step-1--change-the-layout-file
